diff --git a/src/imports/HusMenu.qml b/src/imports/HusMenu.qml
index 6dba48a..95087cd 100644
--- a/src/imports/HusMenu.qml
+++ b/src/imports/HusMenu.qml
@@ -289,6 +289,11 @@ Item {
         __listView.model = [];
     }
 
+    function clearSelectedMenu() {
+        __private.gotoMenuKey = '';
+        __private.selectedItem = null;
+    }
+
     component MenuButton: HusButton {
         id: __menuButtonImpl
 
